Shoot/www/js/app.js

```diff
@@ -20,17 +22,19 @@ angular.module('starter', ['ionic', 'starter.controllers', 'starter.services'])
     oauth2.addKeycloak({
       name: 'keycloak',
       settings: {
-        base: 'http://192.168.0.12:8080/auth',
+        base: '<location of keycloak server e.g. http://192.168.0.12:8080/auth>',
+        //when you use the shoot and share backend these do not need to change
         clientId: 'shoot-third-party',
-        realm: "shoot-realm"
+        realm: 'shoot-realm'
       }
     });
     
     oauth2.addFacebook({
       name: 'facebook',
       settings: {
-        clientId: '1511044619160050',
-        clientSecret: '3b08052d3d96e2120f2c53a36eebd02f',
+        //be sure to also update the <project_name>-Info.plist file with the clientId
```

Comment:
The `Info.plist` hint is for iOS only, perhaps we add something like: "On iOS, be sure..."